### PR TITLE
add build recipe for flux-core

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

**Comment by vchuravy** [comment]
Seems like my struggles with Lua are not yet over.

**Comment by giordano** [comment]
They don't seem to believe in 32 bits?

File: F/flux_core/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    Dependency("Jansson_jll"),
+    Dependency("Libuuid_jll"),
+    Dependency("Lz4_jll"),
+    Dependency("Hwloc_jll"),
+    Dependency("SQLite_jll"),
+    Dependency("Lua_jll"),
+    HostBuildDependency("Lua_jll"),
+]
+
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```
Comment (giordano) [inline F/flux_core/build_tarballs.jl:78]:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)

```

**Comment by vchuravy** [comment]
No I will add that to my fixes for upstream

**Pull request merged**